### PR TITLE
remotetool: add command to upload a blob into the CAS

### PR DESCRIPTION
**Pull request by Yannic**

### CODE REVIEW

File: go/pkg/tool/tool_test.go [hunk 1]
```diff
@@ -206,3 +208,43 @@ func TestTool_DownloadBlob(t *testing.T) {
...
+func TestTool_UploadBlob(t *testing.T) {
+	e, cleanup := fakes.NewTestEnv(t)
+	defer cleanup()
+	cas := e.Server.CAS
+
+	tmpFile, err := ioutil.TempFile(t.TempDir(), "")
+	if err != nil {
+		t.Fatalf("TempFile failed: %v", err)
+	}
+	if _, err := io.WriteString(tmpFile, "Hello, World!"); err != nil {
```
Comment (ola-rozenfeld) [inline go/pkg/tool/tool_test.go:221]: nit: use ioutil.WriteFile instead, it's shorter.

File: go/cmd/remotetool/main.go [hunk 2]
```diff
@@ -115,6 +117,14 @@ func main() {
 			log.Exitf("error reexecuting action %v: %v", *digest, err)
 		}
 
+	case uploadBlob:
+		if *pathPrefix == "" {
+			log.Exitf("--path must be specified.")
+		}
+		if err := c.UploadBlob(ctx, *digest, *pathPrefix); err != nil {
```
Comment (ola-rozenfeld) [inline go/cmd/remotetool/main.go:124]: Don't pass the digest from here, compute and print it from the file contents inside the function.

File: go/pkg/tool/tool.go [hunk 3]
```diff
@@ -216,6 +217,21 @@ func (c *Client) DownloadBlob(ctx context.Context, blobDigest, path string) (str
 	return string(contents), nil
 }
 
+// UploadBlob uploads a blob from the specified path into the remote cache.
+func (c *Client) UploadBlob(ctx context.Context, blobDigest, path string) error {
+	dg, err := digest.NewFromString(blobDigest)
```
Comment (ola-rozenfeld) [inline go/pkg/tool/tool.go:222]: Use NewFromFile here to compute the digest from the path.

File: go/cmd/remotetool/main.go (hunk 2 above)
Comment (Yannic) [reply go/cmd/remotetool/main.go:124]: done

File: go/pkg/tool/tool.go (hunk 3 above)
Comment (Yannic) [reply go/pkg/tool/tool.go:222]: done

File: go/pkg/tool/tool_test.go (hunk 1 above)
Comment (Yannic) [reply go/pkg/tool/tool_test.go:221]: done

**Review by Yannic — COMMENTED** [review]
Thanks, done! PTAL

**Comment by ola-rozenfeld** [comment]
I think you forgot to push the new commits.

**Comment by Yannic** [comment]
Indeed... it's pushed now.

**Review by ola-rozenfeld — APPROVED** [review]

**Pull request merged**